src/languages/types.ts

```diff
@@ -190,7 +192,41 @@ type RemovedTheRequestParams = {valueName: string; oldValueToDisplay: string};
 
 type UpdatedTheRequestParams = {valueName: string; newValueToDisplay: string; oldValueToDisplay: string};
 
+/* Translation Object types */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type TranslationBaseValue = string | string[] | ((...args: any[]) => string);
```

Comment:
@blazejkustra We need `any[]` here I think, I tried with `unknown[]` and every function starts failing.